copilot/fastapi/app/tvp.py

```
@@ -0,0 +1,392 @@
+    If a thread is already running, it will not start a new one.
+    """
+    global reconciliation_thread
+    
+    if reconciliation_thread and reconciliation_thread.is_alive():
+        return
+    
+    def periodic_reconcile() -> None:
+        while True:
+            try:
+                reconcile_from_git()
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)